src/app/technical-record/technical-record.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="govuk-width-container">
-  <a class="govuk-back-link" vtmBackButton>Back</a>
+  <a class="govuk-back-link" vtmBackButton url="search">Back</a>
```

Comment:
We might want to add an id here for testers, we will have to check.

Reply:
reverted to old implementation